website/public/js/controllers/sortableInventoryCtrl.js

```diff
@@ -0,0 +1,14 @@
+habitrpg.controller('SortableInventoryController', ['$scope',
+  function ($scope) {
+
+    // doing this in a function causes 10 $digest iteration error
+    $scope.attributeSort = {};
+    $scope.attributeSort[env.t('con')] = ['-con', '-(con+int+per+str)'];
```

Comment:
I still don't understand why this needs to be using the translated versions of the keys. I'd think that would be a wonky way to assign keys to an object since we can't fully control what those values are. IE, if someone made a mistake in a translation and used the same word for `con` as `int`, this would break.